network/manager.go

```diff
@@ -214,12 +214,6 @@ func (nm *networkManager) restore(isRehydrationRequired bool) error {
 	}
 
 	log.Printf("[net] Restored state")
-	for _, extIf := range nm.ExternalInterfaces {
```

Comment:
Why do we need to remove this part ? Is this not used anymore?
My understanding is that number of endpoints after restoring the state is displayed in the logs.

Reply:
moved to a place where we create and delete endpoints.. this is causing spam for windows mulitenancy case where customer create network per vlan id and this log getting looped for all networks each time on add/del